src/lib.rs

```diff
@@ -42,6 +42,20 @@ fn into_backend(db: DB, api: GoApi, querier: GoQuerier) -> Backend<GoApi, GoStor
     }
 }
 
+#[repr(C)]
+#[derive(Copy, Clone, Default)]
+pub struct AnalysisReport {
+    pub has_ibc_entry_points: bool,
+}
+
+impl From<cosmwasm_vm::AnalysisReport> for AnalysisReport {
+    fn from(report: cosmwasm_vm::AnalysisReport) -> Self {
+        AnalysisReport {
+            has_ibc_entry_points: report.has_ibc_entry_points,
+        }
+    }
+}
```

Comment:
Nice implementation. I'm glad we don't need to worry about C layout in `cosmwasm_vm`.

Reply:
Yeah, I'd only do with that right at the FFI boundary.

Normal rust types in cosmwasm-vm and normal go types in wasmd.
This type just is used inside this same repo on both sides of FFI